backend/Bindings/Edna.Bindings.LtiAdvantage/Utils/EdnaExternalHttpHandlerExtension.cs

```diff
@@ -0,0 +1,17 @@
+﻿// --------------------------------------------------------------------------------------------
```

Comment:
I'm seeing a similar file in Utils folder as well. Any reason for this duplication 🤔 ?

Reply:
> The build seems to be failing. Please take a look.

Yes, I have taken care of the same. It's working fine now.

Reply:
> I'm seeing a similar file in Utils folder as well. Any reason for this duplication 🤔 ?

Ideally, they all must only be in the Utils folder only. However, due to a package version conflict (mentioned above), I had to add a duplicate version of the changes. I will right away work on a follow-up task wherein we upgrade the package version in LTIAdvantage, and remove the redundancy.